common/v2/components/WalletUnlock/Web3ProviderInstall.tsx

```diff
@@ -72,6 +73,10 @@ function InstallMetaMask() {
 }
 
 function Web3ProviderInstall() {
+  useEffect(() => {
+    AnalyticsService.instance.track(ANALYTICS_CATEGORIES.ADD_WEB3_ACCOUNT, `No provider detected`);
+  }, [])
```

Comment:
How about a custom hook for analytics? Something we can use like:
```
const { track } = useAnalytics()
useFirstMount(() => { track(ANALYTICS_CATEGORIES.ADD_WEB3_ACCOUNT, 'No provider detected' )})
```

Reply:
This sounds like a solid idea - we would be able to replace all current calls with this approach, right?

Reply:
As Povio will be picking up the remaining analytics tasks after this PR is merged, do you think it would be wise to assign this as a separate task to them? Preferably, I would like to have this PR merged ASAP, so Povio can start working on the rest :+1:

Reply:
Ok, I'll merge now then